docs/kyma-environment-broker/03-09-subaccount-cleanup-cronjob.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Subaccount Cleanup CronJob
+type: Details
+---
+
+Each SKR instance in Kyma Environment Broker (KEB) database belongs to a global account and to a subaccount.
+Subaccount Cleanup is an application which periodically calls the CIS service and notifies about SUBACCOUNT_DELETE events. 
+Based on these events, Subaccount Cleanup triggers the deprovisioning action on the SKR instance to which a given subaccount belongs.
+
+## Details
+
+The Subaccount Cleanup process is divided into several steps:
+
+1. Fetch SUBACCOUNT_DELETE events from the CIS service.
+
+    a. CIS client makes a call to the CIS service and as a response, it gets a list of events divided into pages. 
+
+    b. CIS client fetches the rest of the events by making a call to each page one by one.
+
+    c. A subaccount is taken from each event and kept in an array.
+
+    d. When the CIS client ends its workflow, it displays logs with information on how many subaccounts were fetched.
+
+2. Find all instances in the KEB database based on the fetched subaccounts.
+
+    a. The subaccounts pool is divided into pieces. For each piece, a query is made to the database to fetch instances.
```

Comment:
```suggestion
>**NOTE:** The subaccounts pool is divided into pieces. For each piece, a query is made to the database to fetch instances.
```

Reply:
... or just join it with the previous sentence. There's no sense in making a one-bullet list.